src/main/java/org/usfirst/frc4904/robot/commands/IntakeRollersIntake.java

```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.CargoIO;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+
+/**
+ * IntakeRollersIntake is a command
+ * that makes the cargo intake rollers
+ * spin inwards to intake the cargo
+ */
+
+public class IntakeRollersIntake extends CommandGroup {
```

Comment:
I think the best way to approach the intake, outtake, and release commandgroups is to make one commandgroup called cargoIORollersSet that takes in a double speed as a parameter, and sets both motors to the speed, then having a cargoIOIntake, cargoIOOuttake, and cargoIORelease extend cargoIORollersSet